staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -358,15 +358,15 @@ func TestPatchResourceNumberConversion(t *testing.T) {
 	tc.startingPod.Namespace = namespace
 	tc.startingPod.UID = uid
 	tc.startingPod.ResourceVersion = "1"
-	tc.startingPod.APIVersion = examplev1.SchemeGroupVersion.String()
+	//tc.startingPod.APIVersion = examplev1.SchemeGroupVersion.String()
```

Comment:
Can you remove this line entirely (instead of commenting it out)?

Just to confirm, this panics [here](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go#L151) because now the object is converted to `*v1.Pod` from `*example.Pod`?

Reply:
Done.

Just to confirm - with this line, it panics here because now the object is converted to *v1.Pod from *example.Pod?
Yes.